service/rolesService.ts

```diff
@@ -0,0 +1,15 @@
+const axios = require('axios');
+import { Roles } from "../model/Roles";
+axios.defaults.baseURL = process.env.API_URL;
+
+module.exports.URL = '/job-roles'
+
+module.exports.getJobRoles = async function(): Promise<Roles[]>{
+    try{
+        const response =await axios.get(this.URL)
+        
+        return response.data
+    }catch (e) {
+        throw new Error('Could not get job roles')
```

Comment:
What HTTP status code would be returned here?

Reply:
Depending on the issue a 404 or 500, would you like me to add a way in to differentiate the two via the error.

Reply:
Yeah, if you could differentiate between then that would be good

Reply:
Added Changes to allow for this